test/unit/lib/otel/consumer.test.js

```diff
@@ -59,8 +59,8 @@ test('should create consumer segment from otel span', (t) => {
   const { segment, transaction } = synth.synthesize(span)
   assert.equal(segment.name, expectedName)
   assert.equal(segment.parentId, segment.root.id)
-  assert.equal(transaction.name, expectedName)
-  assert.equal(transaction.type, 'bg')
+  assert.equal(transaction.name.endsWith(expectedName), true)
```

Comment:
this name should match exactly. the only thing you'll have to do before this, for now, is end the transaction. Once the reconciliation ticket is done(#2932) when the consumer span ends it will end the transsaction